core-common/src/main/kotlin/io/nlopez/rules/core/util/Composables.kt

```diff
@@ -62,8 +62,8 @@ private val KtCallExpression.containsComposablesWithModifiers: Boolean
 
         // Check if there is any Modifier chain (e.g. `Modifier.fillMaxWidth()`)
         return valueArguments.mapNotNull { it.getArgumentExpression() }
-            .flatMap { it.findChildrenByClass<KtReferenceExpression>() }
```

Comment:
This was the cause of the issues: this was making it so the check for `Modifier` would leak in the cases where a slot param was used. Any usage of Modifier in subsequent child components would trigger this and mark it as a UI emitter composable.